doc/guide/writing-persistent-objects.rst

```diff
@@ -583,8 +585,12 @@ inherits identity-based ``__eq__`` and ``__hash__``::
      def add_author(self, author):
          self.authors += (author, )
 
-Lets see an example of how these classes behave when stored in a
-dictionary. First, lets store some dictionaries::
+.. -> src
```

Comment:
End the previous sentence with a period and move the class into the proof. The reader doesn't need to see the class that *doesn't* define __eq__ and __hash__. :)